build/do.rq

```diff
@@ -163,6 +184,134 @@ write_readme {
 	run("./regal table --write-to-readme bundle")
 }
 
+fetch_engine_caps {
+	fetch_eopa_caps
+}
+
+fetch_eopa_caps {
+
+	# git ls-remote --tags output looks like this:
+	#
+	#	...
+	#	bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38        refs/tags/v1.0.0
+	#	bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38        refs/tags/v1.0.1
+	#	9ace37dda9859a1afd2fcd6a87dd7f1955be4dc4        refs/tags/v1.10.1
+	#	...
+	#
+	# so .stdout will look like this:
+	#
+	#	...
+	#	[
+	#		"bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38\trefs",
+	#		"tags",
+	#		"v1.0.0"
+	#	],
+	#	[
+	#		"bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38\trefs",
+	#		"tags",
+	#		"v1.0.1"
+	#	],
+	#	[
+	#		"bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38\trefs",
+	#		"tags",
+	#		"v1.1.0"
+	#	],
+	#	...
+
+	print("fetching tags for enterprise-opa repository")
+
+	eopa_tags_result := rq.run([
+			"git",
+			"ls-remote",
+			"--tags",
+			"https://github.com/styrainc/enterprise-opa"
+		], {
+			"stdout_spec": {
+				"format": "raw",
+				"options": {
+					"raw.fs": "/",
+					"raw.rs": "[\n\r]"
+				},
+			}
+		}
+	)
+
+	{ rq.error(sprintf("\nstdout: %s\nstderr: %s\n", [eopa_tags_result.stdout, eopa_tags_result.stderr])) | eopa_tags_result.exitcode != 0 }
+
+
+	# We assume that tags and capabilities files are 1:1, but some EOPA
+	# release tags in the past did not correctly get capabilities files, so
+	# we eliminate them from consideration.
+	
+	known_bad_tags := {
+		"v0.100.5",  # tag missing capabilities file
+		"v0.100.6",  # tag missing capabilities file
+		"v0.100.7",  # tag missing capabilities file
+		"v0.49.0-5", # tag missing capabilities file
+		"v0.49.0-6", # tag missing capabilities file
+		"v0.49.0-7", # tag missing capabilities file
+		"v0.49.0-8", # tag missing capabilities file
+		"v1.15.0",   # tag missing capabilities file (misnamed v0.15.0)
+		"v1.4.1",    # tag missing capabilities file
+		"v1.5.0",    # tag missing capabilities file
+	}
+
+	eopa_tags := {t | r := eopa_tags_result.stdout[_]; t := r[2] ; not known_bad_tags[t]}
```

Comment:
I don't think we need the entire history of EOPA caps -- what for? As an arbitrary cut-off, I think I'd drop everything pre-1.0.0.

Reply:
9d500efb5ceac61864441fd608918e5362f2ae1b

Reply:
Thanks!